docs/development-guide.md

```diff
@@ -17,6 +17,13 @@ it is **highly** encouraged to:
    [install](https://golang.org/doc/install) it on your system.
 * Setup the [GOPATH](http://www.g33knotes.org/2014/07/60-second-count-down-to-go.html)
    environment.
+* Ceph-CSI uses the native Ceph libaries through the [go-ceph
+   package](https://github.com/ceph/go-ceph). It is required to install the
+   Ceph C headers in order to compile Ceph-CSI. The packages are called
```

Comment:
`Ceph-CSI` to `ceph-csi` as  this is not header

Reply:
Really? Ceph is a name, and CSI is an abbreviation. The main README also uses "Ceph CSI" (sometimes with the "-", sometimes without).

Reply:
yeah, in lot of places we are using ceph-csi in few places we are calling Ceph-CSI